core/src/components/input/input.ionic.outline.scss

```diff
@@ -0,0 +1,14 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline.input-shape-round) .input-bottom,
+:host(.input-fill-outline.input-label-placement-floating) .input-bottom {
+  /**
+   * The bottom content should take on the start and end
+   * padding so it is always aligned with either the label
+   * or the start of the text input.
+   */
+  @include padding-horizontal(var(--padding-start), var(--padding-end));
+}
```

Comment:
While I was implementing the valid state, I noticed that some styles were missing from the helper text.